data/json/items/gun/4570.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "id": "1895sbl",
+    "copy-from": "rifle_manual",
+    "type": "GUN",
+    "name": "Marlin 1895 SBL",
+    "description": "A handy but powerful lever-action rifle chambered for .45-70 Government. Designed for wilderness guides for defense against large predators such as grizzly bears, moose, and dinosaurs.",
+    "weight": 3650,
+    "volume": 9,
+    "price": 400000,
```

Comment:
$4000?

Reply:
I don't really have a good frame of reference for weapon "prices", they seem so all over the place.  Feel free to make a more reasonable number.

Reply:
<https://www.cabelas.com/product/Marlin-Model-Big-Bore-Lever-Action-Rifles/706393.uts#productChart>

$700.

Reply:
The 1895 SBL model in the same link was $1000.  Prices updated.

Reply:
Since you've put it in the '    "id": "guns_rifle_common", group, I'd use the cheapest or at least middle of the road model. Why the SBL model specifically? The other ones would be more common.

Reply:
It's possibly the most desirable and iconic model due to appearing in a few movies, namely Jurassic World.  Marlin is selling every single one they make .  I've never seen one sit on a shelf for more than a week for the past few years.  I've technically been waiting for one from my local gun store for about 2-3 years, but I eventually broke down and got the Henry All Weather.

It also has the XS rail for more believable mounting of optics.

While models like the 1895G might be cheaper and more obtainable, perhaps having these rifles be too common isn't the best idea, considering the damage output?

I had it in rare, but I moved it to common with the same rarity as the .300 Win Mag Model 70 after I didn't encounter it a single time in testing while it was in rare.  I can always move it back.